src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -938,7 +940,8 @@ Box2f CropWindowTool::resolutionGate() const
 	{
 		if( const ImagePlug *in = imageView->inPlug<ImagePlug>() )
 		{
-			resolutionGate = Box2f( V2f( 0 ), V2f( in->format().width(), in->format().height() ) );
+			const Format format = in->format();
```

Comment:
Again, we need a context. I wonder if the most pragmatic course of action is to make `ImageGadget::format()` public, since that already does the right thing and tracks dirtiness internally to avoid unnecessary recomputes.

Reply:
https://github.com/GafferHQ/gaffer/pull/3691#discussion_r398384956 Addresses this in a non-API changing way so we can do it in `56_maintenance`?

Reply:
As discussed separately, adding the function I suggested wouldn't have broken ABI and would have been fine in `56_maintenance`. But this fix is fine too...